src/App.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect } from "react";
+import "./App.css";
+import axios from "axios";
+
+//image
+import shoesCOMME from "./img/shoesCOMME.png";
+import shoesFire from "./img/shoesFIre.png";
+import shoesBlack from "./img/shoesBlack.png";
+import shoesFireBlack from "./img/shoesFireBlack.png";
+import shoesCamo from "./img/shoesCamo.png";
+import shoesGreen from "./img/shoesGreen.png";
+import shoesBlue from "./img/shoesBlues.png";
+import shoesCons from "./img/shoesCons.png";
+import shoesRed from "./img/shoesRed.png";
+import shoesRed01 from "./img/shoesRed01.png";
+import shoesPink01 from "./img/shoesPink01.png";
+import shoes01 from "./img/shoes01.png";
+import shoesBas from "./img/shoesbas.png";
+import shoesGaru from "./img/shoesGaru.png";
+import shoes03 from "./img/shoes03.png";
+
+//Components
+import Allshoes from "./allshoes";
+import White from "./White";
+
+export default function App() {
+  const [allshoesName, setallShoesName] = useState([]);
+  const [yearView, setYearView] = useState(false);
+  const allImage = {
+    1: shoesBlack,
+    2: shoesFire,
+    3: shoesRed01,
+    4: shoesGreen,
+    5: shoes01,
+    6: shoesBlue,
+    7: shoesCOMME,
+    8: shoesFireBlack,
+    9: shoesGaru,
+    10: shoesBas,
+    11: shoesCamo,
+    12: shoesBlue,
+    13: shoesBlack,
+    14: shoesPink01,
+    15: shoesFireBlack,
+    16: shoes03,
+  };
+
+  useEffect(() => {
+    async function getData() {
+      let res = await axios.get("api/allshoes");
+      // const result = [...allshoesName, ...res.data];
+      console.log("getting data!");
+      // console.log(res.data);
+      // let result = [...allshoesName, ...res.data];
+      // setState can take fucntion, parameter is provious value return next value
+      setallShoesName((prevAllShoesname) => [...prevAllShoesname, ...res.data]);
+    }
+    getData();
+  }, []);
+
+  return (
+    <div>
+      <header className="header">
+        <h1>SHOES RACK</h1>
+        <div className="menu">
+          <div
```

Comment:
Instead of `div` with `onClick` it's better to use `button` element for [accessibility](https://developer.mozilla.org/ja/docs/Learn/Accessibility/What_is_accessibility), and also so it's clearer from your code that it should be clickable.